lib/matplotlib/axes/_base.py

```diff
@@ -2810,6 +2810,19 @@ def get_xlim(self):
         """
         return tuple(self.viewLim.intervalx)
 
+    def _validate_axis_limits(self, limit, convert):
+        """
+        Raise ValueError if specified axis limits are infinite.
+
```

Comment:
The method name is a bit misleading. It validates the converted limit. Also, the docstring doesn't say that it returns anything useful, nor that None can be an input.